defaults/main.yml

```diff
@@ -2,5 +2,5 @@
 # Pin-Priority of NodeSource repository
 nodejs_nodesource_pin_priority: 500
 
-# 0.10 or 0.12 or 4.x
-nodejs_version: "4.2"
+# 0.10 or 0.12 and 4.x, 5.x and onward
+nodejs_version: "5.0"
```

Comment:
I _think_ it might be wise to pin this to 4.x rather than 5.x because those who use Ansible are more likely to be LTS type users and v5.x is only going to last a max of 8 months so we'll be updating this config pretty soon.